Content.Server/Speech/EntitySystems/OwOAccentSystem.cs

```diff
@@ -1,41 +1,175 @@
 using Content.Server.Speech.Components;
-using Robust.Shared.Random;
+using System.Text.RegularExpressions;
 
 namespace Content.Server.Speech.EntitySystems
 {
     public sealed class OwOAccentSystem : EntitySystem
     {
-        [Dependency] private readonly IRobustRandom _random = default!;
-
-        private static readonly IReadOnlyList<string> Faces = new List<string>{
-            " (・`ω´・)", " ;;w;;", " owo", " UwU", " >w<", " ^w^"
-        }.AsReadOnly();
-
-        private static readonly IReadOnlyDictionary<string, string> SpecialWords = new Dictionary<string, string>()
-        {
-            { "you", "wu" },
-        };
+        // Dictionary to hold words to be replaced.
+        private static Dictionary<string, string> SpecialWords = new Dictionary<string, string>();
```

Comment:
In net 6 and so on, you can define type in declaration.
```suggestion
        private static Dictionary<string, string> SpecialWords = new ();
```